src/webgl/loading.js

```diff
@@ -177,6 +177,94 @@ p5.prototype.loadModel = function(path) {
   }
   return model;
 };
+/**
+ * @method loadModel
+ * @param  {String} path
+ * @param  {Object} [options]
+ * @param  {function(p5.Geometry)} [options.successCallback]
+ * @param  {function(Event)} [options.failureCallback]
+ * @param  {String} [options.fileType]
+ * @param  {boolean} [options.normalize]
+ * @return {p5.Geometry} the <a href="#/p5.Geometry">p5.Geometry</a> object
+ */
+p5.prototype.loadModel = function(path,options) {
+  p5._validateParameters('loadModel', arguments);
+  let normalize= false;
+  let successCallback;
+  let failureCallback;
+  let fileType = path.slice(-4);
+  if (options && typeof options === 'object') {
```

Comment:
This if/else structure looks good! Now that we have a version where we can pass in an object, did we also want to add optional `flipU` and `flipV` properties to the object? Where, if they are set to true, before calling the success callback, we would call `.flipU()` and/or `.flipV()` on the model?

Reply:
These properties are now added, do they look okay?